src/components/common/Dialogs/CreatePaymentDialog/validation.ts

```diff
@@ -39,7 +39,7 @@ const validationSchema = object()
         'more-than-zero',
         () => MSG.amountZero,
         (value) => {
-          const numberWithoutCommas = (value || '0').replace(/,/g, '');
+          const numberWithoutCommas = (value || '0').replace(/,/g, ''); // @TODO: Remove this once the fix for FormattedInputComponent value is introduced.
```

Comment:
Do you mean the downgrading of `cleave.js`? Nope, this is related to this fix that @jakubcolony implemented: https://github.com/JoinColony/colonyCDapp/pull/320/commits/dd6f97d886bb3207d2c29018a6ebdd85544e86ae

Basically, the cleave was using the formatted value of the input instead of the raw value like in the Dapp. So, we would get errors when dealing with the value stored by the form because they would have commas `,`. This is a solution I implemented before Jakub informed me of the issue with the cleave value.

I should remove this line when Jakub's fix makes its way into the branch.